watertap/property_models/NaCl_T_dep_prop_pack.py

```diff
@@ -0,0 +1,1791 @@
+            doc="Mass density parameter A4",
+        )
+        self.dens_mass_param_A5 = Var(
+            within=Reals,
+            initialize=-4.657e-8,
+            units=dens_units * t_inv_units**4,
+            doc="Mass density parameter A5",
+        )
+
+        # TODO: add vapor phase and according properties (if deemed necessary)
+        # including: specific enthalpy, diffusivity(water-air), density, specific heat, heat of vaporization
+
+        # traditional parameters are the only Vars currently on the block and should be fixed
+        for v in self.component_objects(Var):
+            v.fix()
+
+        # ---default scaling---
+        self.set_default_scaling("temperature", 1e-2)
+        self.set_default_scaling("pressure", 1e-6)
+        self.set_default_scaling("dens_mass_phase", 1e-3, index="Liq")
+        self.set_default_scaling("visc_d_phase", 1e3, index="Liq")
+        self.set_default_scaling("diffus_phase_comp", 1e9, index=("Liq", "NaCl"))
+        self.set_default_scaling("osm_coeff", 1e0)
+        self.set_default_scaling("enth_mass_phase", 1e-2, index="Liq")
+        self.set_default_scaling("cp_mass_phase", 1e-4, index="Liq")
+        self.set_default_scaling("vapor_pressure", 1e2)
+        self.set_default_scaling("th_cond_phase", 1e0, index="Liq")
+        self.set_default_scaling("solubility", 1e0)
+
+    @classmethod
+    def define_metadata(cls, obj):
+        """Define properties supported and units."""
+        obj.add_properties(
+            {
+                "flow_mass_phase_comp": {"method": None},
+                "temperature": {"method": None},
+                "pressure": {"method": None},
+                "mass_frac_phase_comp": {"method": "_mass_frac_phase_comp"},
+                "dens_mass_phase": {"method": "_dens_mass_phase"},
+                "flow_vol_phase": {"method": "_flow_vol_phase"},
+                "flow_vol": {"method": "_flow_vol"},
+                "conc_mass_phase_comp": {"method": "_conc_mass_phase_comp"},
+                "flow_mol_phase_comp": {"method": "_flow_mol_phase_comp"},
+                "mole_frac_phase_comp": {"method": "_mole_frac_phase_comp"},
+                "molality_phase_comp": {"method": "_molality_phase_comp"},
+                "diffus_phase_comp": {"method": "_diffus_phase_comp"},
+                "visc_d_phase": {"method": "_visc_d_phase"},
+                "pressure_osm_phase": {"method": "_pressure_osm_phase"},
+                "enth_mass_phase": {"method": "_enth_mass_phase"},
+                "cp_mass_phase": {"method": "_cp_mass_phase"},
+            }
+        )
+        obj.define_custom_properties(
+            {
+                "osm_coeff": {"method": "_osm_coeff"},
+                "enth_flow": {"method": "_enth_flow"},
+                "solubility": {"method": "_solubility"},
+                "th_cond_phase": {"method": "_th_cond_phase"},
+                "vapor_pressure": {"method": "_vapor_pressure"},
```

Comment:
Can double check naming conventions in IDAES to see if anything can be refined here. Also, for consistency, may want to add qualifiers (i.e. `solubility_comp` where the component is NaCl, even though it is the only solute)

Reply:
1. Anything that is indexed should have the indexing set(s) as a suffix
2. I think this is `therm_cond_phase`
3. `pressure_sat`

Reply:
Thank you for bringing this to my attention! I was unaware of the naming conventions in the IDAES documentation. The naming is now updated.